### DISCUSSION TITLE
The weekday display problem with date picker

### DISCUSSION THREAD

**Discussion by zhiiwg** [firefly-iii/firefly-iii#6616] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

the week day of date picker in new withdrawal page is not correct.
2022-11-07 is monday, but date picker shows 2022-11-07 is sunday.
![image](https://user-images.githubusercontent.com/4954007/200205907-6e4df46a-e13d-4708-8b1c-d2f8bd1a4798.png)
![image](https://user-images.githubusercontent.com/4954007/200205970-e391219c-2415-4ab1-9a5f-21d607dc9925.png)

same in demo site.
![image](https://user-images.githubusercontent.com/4954007/200206072-992698bc-5495-4c3f-bb1a-75d0f01b1f02.png)

### Expected behaviour

date picker shows 2022-11-07 is monday.

### Debug information

Debug information generated at 2022-11-07 08:57:35 Asia/Shanghai for Firefly III version **5.7.15**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.7.15 |
| Firefly III API | 1.5.6 |
| Build | 298, 02-11-2022 05:00:04 CET |
| DB version | 18 (exp. 18) |
| PHP | `8.1.11` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Asia/Shanghai |
| Browser TZ | Asia/Shanghai |
| App environment | production |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | zh_CN |
| Default locale | equal |
| BCscale | 24 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| Trusted proxies (.env) | `**` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start | 2022-11-01 00:00:00 |
| Session end | 2022-11-30 23:59:59 |
| Session first | 2021-03-01 00:00:00 |
| User ID | 1 |
| User language | zh_CN |
| User locale | zh_CN |
| Attempt at "zh_CN.utf8" | 'zh_CN.utf8' |
| Attempt at "zh_CN.UTF-8" | 'zh_CN.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.84 Safari/537.36 HBPC/12.1.1.301 |

### Steps to reproduce

1. go to firefly-iii site
2. click new withdrawal
3. click date picker
4. check if the weekday is correct

**Comment by JC5** [comment, marked as answer]
The date dropdown is rendered by your browser, not by Firefly III. I just set `type="date"`.

**Reply by zhiiwg** [reply]
OK, I tried Firefox and Edge, and the date picker shows correct weekdays.
Maybe I should report this problem to the browser dev team.
Thanks!
